packages/host/app/services/matrix-sdk-loader.ts

```diff
@@ -204,6 +270,10 @@ function extendedClient(
     get(target, key, receiver) {
       let extendedTarget = target as unknown as ExtendedClient;
       switch (key) {
+        case 'setAccountData':
+          return setAccountData.bind(extendedTarget, fetch);
+        case 'getAccountData':
```

Comment:
This will shadow the actual method with this name on matrix-js-sdk's client: https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/client.ts#L4232

Reply:
I added this, because despite being documented, it doesn't actually type check (so i just assumed it was missing). when i run it without these methods added in our code, despite not being typed it does work. so i guess we can just ignore the type errors?

![image](https://github.com/user-attachments/assets/49421b07-be54-4a9a-9cda-abf50d299119)

Reply:
@lukemelia this is fixed now